server/controllers/temporary-accommodation/manage/premisesController.ts

```diff
@@ -27,12 +27,14 @@ export default class PremisesController {
         localAuthorities: allLocalAuthorities,
         characteristics: allCharacteristics,
         probationRegions: allProbationRegions,
+        pdus: allPdus,
```

Comment:
Where is `allPdus` defined in this file?

Reply:
Comes from the call to `this.premisesService.getReferenceData(token)` above. The service felt like the right place to read the JSON, and then the controller doesn't care about the actual source